bin/installer.sh

```diff
@@ -90,6 +91,63 @@ function get_icinga_etc()
 	return 0;
 }
 
+function check_selinux() {
+	# Check SELinux available
+	echo -e "\n==================================";
+	echo "SELinux Check"
+	selinuxStatus=$(getenforce 2>/dev/null)
+
+	if [[ "$selinuxStatus" == "Enforcing" ]] || [[ "$selinuxStatus" == "Permissive" ]]; then
+		echo "SELinux is ${selinuxStatus}"
+		question "Configure SELinux?";
+		if [ $? -eq 0 ]; then return 0; fi
+
+		echo -n "Checking existing SELinux modules: "
+		semodule -l | grep -qw trapdirector
```

Comment:
Add a version test too : 
```
                currentVerion=1.0
		seModuleText = $(semodule -l | grep -qw trapdirector)
```
then after the retcode test, extract version test it and eventually remove old version
```
               version = $(echo $seModuleText | awk -F ' ' '{print $2}' )
               if [ "$version" =="$currentVerion"]; then echo "OK" return 0;
               else semodule -r trapdirector
               fi
```

Reply:
I like this idea. Unfortunately I encountered a road block. It won't be possible due to a change in SELinux: https://access.redhat.com/solutions/2760071

So if you update the module in future, you would have to start blindly reinstalling (the behaviour of `semodule -i trapdirector.pp` is to replace the installed module, regardless of version).